README.md

```diff
@@ -101,6 +101,59 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
+# Development
+
+> This short guide is meant to give a guide to how github apps are made, and basic steps to reproduce the OpenCollective Bot should you wush to fix any bug.
+
+... Note Probot was used to build the Github app hence certain details may be specific to **Probot**
```

Comment:
Which details are specific to Probot?

Reply:
I wanted to write some, information concerning probot but since building a fresh github apps is not among the PR scope I left it out.  

I would remove this line